test/fixtures/outlineTest/test.go

```diff
@@ -6,6 +6,12 @@ import (
 
 var _ string = "foobar"
 
+// const constFoo = "constBar"
+
+type circle struct {
```

Comment:
shouldn't this be interface and not struct?

Reply:
@ramya-rao-a  you're right, just another stupid mistake. already update to my master.